index.js

```diff
@@ -533,7 +535,7 @@ function Physics (mcData, world) {
         // Calculate what the speed is (0.1 if no modification)
         const attributeSpeed = attribute.getAttributeValue(playerSpeedAttribute)
         inertia = (blockSlipperiness[blockUnder.type] || physics.defaultSlipperiness) * 0.91
-        acceleration = attributeSpeed * (0.1627714 / (inertia * inertia * inertia))
+        acceleration = attributeSpeed * (0.16277136 / (inertia * inertia * inertia))
```

Comment:
This is the only actual change in this PR. The other changes have no effect, you can revert them. If there are errors in calculation, they are likely to be caused by 64-bit vs 32-bit float differences, and that's what Math.fround is supposed to fix.